server.go

```diff
@@ -185,6 +199,25 @@ func (p *peerState) forAllPeers(closure func(p *peer)) {
 	p.forAllOutboundPeers(closure)
 }
 
+// handleUpdatePeerHeight updates the heights of all peers who were known to
+// announce a block we recently accepted.
+func (s *server) handleUpdatePeerHeight(state *peerState, umsg updatePeerHeightMsg) {
+	state.forAllPeers(func(p *peer) {
```

Comment:
The source peer, whose new block we just accepted has already had its height updated at this point. The purpose of the function is to update the heights of all other peers who lost the block announcement race. I've pluralized 'Height' in the function to further clarify its behavior.